### PR TITLE
Remove scheule delete from schedule deploy fn

### PR DESCRIPTION
**Pull request by haakonvt**
+ improve logging, before:
```
Warning: schedule: Deleted all (1) existing schedule(s)!
(...)
Warning: function: Function deployment successful! Deployment took 176.48 seconds
Warning: __main__: Successfully created and deployed function example_function1-master with id 8101158008689270
Warning: schedule: No existing schedule(s) to delete!
Warning: schedule: Successfully deployed schedule example_function1-master:My daily schedule with cron expression 0 0 * * *.
```
Now:
```
Warning: schedule: Deleted all (1) existing schedule(s)!
(...)
Warning: function: Function deployment successful! Deployment took 164.39 seconds
Warning: __main__: Successfully created and deployed function example_function1-master with id 6140110941155372
Warning: schedule: Attaching 1 to example_function1-master
Warning: schedule: - Schedule 'example_function1-master:My daily schedule' with cron: '0 0 * * *' attached successfully!

### CODE REVIEW

File: src/schedule.py [hunk 1]
```diff
@@ -24,17 +24,16 @@ def delete_all_schedules_for_ext_id(client: CogniteClient, function_external_id:
...
+    logger.info(f"Attaching {len(config.schedules)} to {function.external_id}")
     for schedule in config.schedules:
         client.functions.schedules.create(
             function_external_id=function.external_id,
             cron_expression=schedule.cron,
             name=schedule.name,
             data=schedule.data,
         )
-        logger.info(f"Successfully deployed schedule {schedule.name} with cron expression {schedule.cron}.")
+        logger.info(f"- Schedule '{schedule.name}' with cron: '{schedule.cron}' deployed successfully!")
```
Comment (haakonvt) [inline src/schedule.py:39]:
```suggestion
        logger.info(f"- Schedule '{schedule.name}' with cron: '{schedule.cron}' attached successfully!")
```

File: src/schedule.py [hunk 2]
```diff
@@ -24,17 +24,16 @@ def delete_all_schedules_for_ext_id(client: CogniteClient, function_external_id:
...
 
 def deploy_schedule(client: CogniteClient, function: Function, config: FunctionConfig):
-    delete_all_schedules_for_ext_id(client, function.external_id)
-
     if not config.schedules:
-        logger.info("Skipped step of attaching schedules!")
+        logger.info("No schedules to attach!")
         return
 
+    logger.info(f"Attaching {len(config.schedules)} to {function.external_id}")
```
Comment (sfsy1) [inline src/schedule.py:31]: might be good to say attaching **schedules** here
```suggestion
    logger.info(f"Attaching {len(config.schedules)} schedules to {function.external_id}")
```

**Review by sfsy1 — APPROVED** [review]
small comment on logging

File: src/schedule.py (hunk 2 above)
Comment (haakonvt) [reply src/schedule.py]: nice ty

**Pull request merged**